src/mongocxx/options/client.cpp

```diff
@@ -21,7 +21,7 @@ MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
 client& client::ssl_opts(ssl ssl_opts) {
-    _ssl_opts = ssl_opts;
+    _ssl_opts = std::move(ssl_opts);
```

Comment:
Coverity pointed out that this was a large parameter by value, which is true. Usually I don't care though because we are moving from the argument. In this case we weren't, so I fixed it.

Reply:
shouldn't this be a const ref?

Reply:
No, it doesn't need to be. We are taking the argument by value because no matter what we are making at least one copy. If we took it by const-ref, we would copy assign into `_ssl_opts`. As written, we get one copy (into the parameter) and one move (into `_ssl_opts`). However, if at the call site the `ssl` object is a temporary, then we actually get two moves and no copies.